.github/workflows/automated-release.yml

```diff
@@ -0,0 +1,67 @@
+name: Automated Release
+on:
+  workflow_dispatch:
+    inputs:
+      releaseBranch:
```

Comment:
Do you plan to cover the creation of the release branch with a (different?) workflow too?